src/chunk.c

```diff
@@ -963,11 +996,11 @@ chunk_find(Hyperspace *hs, Point *p)
 	return chunk;
 }
 
-static bool
+static ChunkResult
 append_chunk_oid(ChunkScanCtx *scanctx, Chunk *chunk)
 {
 	if (!chunk_is_complete(scanctx, chunk))
-		return false;
+		return CHUNK_IGNORED;
```

Comment:
To confirm, we want to change the semantics of `chunk_find_all_oids` to keep appending `OID`s after it finds an incomplete chunk? (So far this is the only semantic change in this PR I've seen)

Reply:
So, the previous return value of `false` here didn't mean "abort" or "done". Note that the semantics of processing chunks in the scan context here are not the same as in the Scanner's tuple processing functions.

Thus, the previous false just meant that we didn't increase `num_processed`, but the processing continued still.  Look at `chunk_scan_ctx_foreach_chunk` above to see that this is the case. 

The difference now is that he semantics are more clear and there is an option to abort processing with `CHUNK_DONE`.